### DISCUSSION TITLE
[Adonis 5] Force table storage engine in migration file

### DISCUSSION THREAD

**Discussion by Melchyore** [adonisjs/core#1414] [Help] [answered]
Hi,

Can we force a storage engine inside migration file?

**Comment by alainrc2005** [comment, marked as answer]
http://knexjs.org/

engine — table.engine(val)
Sets the engine for the database table, only available within a createTable call, and only applicable to MySQL.

I hope this can help you

**Reply by Melchyore** [reply]
@alainrc2005 thanks!
